.travis.yml

```
@@ -15,16 +15,19 @@ services:
   - docker
 
 before_install:
+  - ./scripts/ci/env_travis.sh
```

```suggestion
  - source scripts/ci/env_travis.sh
```

Totally not sure if this works.
There is also the possibility to specify env vars directly in here with yaml syntax.

---

I find it more convinient as a script, because mimicking the `$TRAVIS_*` (because the new `$CI_*` are basically renamed `$TRAVIS_*` for now) vars in github workflows needs some scripting.

---

NOTE:
- **without `source`** - environment variables are not available outside the script
- **with `source`** - environment variables  are available withing the current step
- `echo "::set-env name=CI_VAR::foobar"` - the proper way to set the variable

However, to prevent stupid mistakes, there always shloud be an empty `echo ""` around the `echo "::set-env ..."` lines, because they can get easily broken. For example:

```bash
cat $GITHUB_EVENT_PATH # no newline in the end of the file
echo "::set-env name=CI_VAR::foobar"
```